server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -114,6 +114,7 @@
   private static final Logger logger = LoggerFactory.getLogger(TSServiceImpl.class);
   private static final String INFO_NOT_LOGIN = "{}: Not login.";
   private static final String ERROR_NOT_LOGIN = "Not login";
+  private static final int SUCCESS_CODE = 200;
```

Comment:
I would suggest you move this into ErrorCode and rename it as StatusCode if you prefer the HTTP style things.